geotrek/signage/tests/test_views.py

```diff
@@ -473,22 +474,27 @@ class BladeFilterSetTest(TestCase):
     def setUpTestData(cls):
         cls.model = cls.factory._meta.model
         cls.user = SuperUserFactory.create()
-        cls.signage = SignageFactory()
+
+        cls.manager = OrganismFactory()
+        cls.manager2 = OrganismFactory()
+
+        cls.signage = SignageFactory(manager=cls.manager, code="COUCOU")
+        cls.signage2 = SignageFactory(manager=cls.manager2, code="ADIEU")
+
         cls.blade = cls.factory(signage=cls.signage)
-        cls.signage2 = SignageFactory()
         cls.blade2 = cls.factory(signage=cls.signage2)
 
     def setUp(self):
         self.client.force_login(self.user)
 
     def test_filter_by_organism(self):
-        filter = BladeFilterSet(data={'manager': self.signage.manager})
+        filter = BladeFilterSet(data={'manager': [self.manager.pk,]})
         response = self.client.get(self.model.get_list_url())
 
         self.assertEqual(response.status_code, 200)
         self.assertContains(
             response,
-            f'option value="{self.signage.manager.pk}">{self.signage.manager.organism}</option>'
+            f'option value="{self.manager.pk}">{self.manager.organism}</option>'
```

Comment:
You should test that the signage is in the response (ant that signage2 is not), not the selectors for the filters